deploy/compose/.env.example

```diff
@@ -1,38 +1,61 @@
-HOPRD_CONTAINER_IMAGE=europe-west3-docker.pkg.dev/hoprassociation/docker-images/hoprd:stable
+#### HOPRD Configuration ####
 
+# Hoprd docker image name
+HOPRD_IMAGE=europe-west3-docker.pkg.dev/hoprassociation/docker-images/hoprd:stable
+
+# Hoprd node memory request and limit
 HOPRD_MEM_REQUEST=1g
 HOPRD_MEM_LIMIT=2g
 
+# Hoprd ports. The P2P port is the one that the node uses to communicate with other nodes and needs to be open to the internet
 HOPRD_API_PORT=3001
 HOPRD_P2P_PORT=9001
 
+# Hoprd log level and max size
 HOPRD_LOG_LEVEL=info
-
 HOPRD_LOG_MAX_SIZE=1000M
 
-HOPR_ADMIN_CONTAINER_IMAGE=europe-west3-docker.pkg.dev/hoprassociation/docker-images/hopr-admin:stable
+#### HOPR Admin Tool Configuration ####
+
+# Hopr Admin tool image name
+HOPR_ADMIN_IMAGE=europe-west3-docker.pkg.dev/hoprassociation/docker-images/hopr-admin:stable
 
-HOPR_ADMIN_PORT=4677
+# Hopr Admin tool port
+HOPR_ADMIN_PORT=8080
 
+# Hopr Admin tool memory request and limit
+HOPR_ADMIN_MEM_REQUEST=256M
 HOPR_ADMIN_MEM_LIMIT=256M
 
-METRICS_CADVISOR_IMAGE=gcr.io/cadvisor/cadvisor:v0.47.2@sha256:78cf73632de30193dc249923fdfdb1d43ee7cfccbf64662991de89ff7c4b1fda
+#### Monitoring Configuration ####
+
+# cAdvisor image name
+METRICS_CADVISOR_IMAGE=gcr.io/cadvisor/cadvisor:v0.49.1
+
+# cAdvisor listening port
 METRICS_CADVISOR_PORT=9093
 
-METRICS_NODE_EXPORTER_IMAGE=prom/node-exporter:v1.7.0@sha256:4cb2b9019f1757be8482419002cb7afe028fdba35d47958829e4cfeaf6246d80
+# Node exporter image name
+METRICS_NODE_EXPORTER_IMAGE=prom/node-exporter:v1.7.0
```

Comment:
The hashes were intentional version pin.

Reply:
But they were not working with different platforms

Reply:
Neither will the rest of the others, yet you left them there, right?